inc/patterns/general-404.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * 404 content.
+ */
+return array(
+	'title'      => __( '404 content', 'twentytwentytwo' ),
+	'categories' => array( 'twentytwentytwo-general' ),
+	'content'    => '<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"200","lineHeight":"1","spacing":{"margin":{"top":"0","bottom":"0"}}}},"className":"has-text-align-center"} -->
```

Comment:
I had to zero the top and bottom margins here because the markup changed slightly when inserting it as a block.